HappyCamps/src/app/services/api.service.ts

```diff
@@ -0,0 +1,17 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ApiService {
```

Comment:
Please rename this service to be more specific (UserService w/e)

Reply:
This was created only for testing purpose only.  Will be deleted in the upcoming commits.